frontend/cypress/e2e/beta/draft-and-review-release.cy.ts

```diff
@@ -0,0 +1,94 @@
+const baseURLForRelease = '/beta/'
+let modelUuidForRelease = ''
+const modelNameForRelease = 'Test Model'
+const releaseVersion = '1.0.0'
+
+describe('Draft and review a model release', () => {
+  before(() => {
+    cy.log('Upload new model and set schema via API')
+    cy.request('POST', 'http://localhost:8080/api/v2/models', {
+      name: modelNameForRelease,
+      teamId: 'Uncategorised',
+      description: 'This is a test',
+      visibility: 'public',
+    }).then((response) => {
+      expect(response.status).to.eq(200)
+      expect(response.body.model).to.have.property('name', modelNameForRelease)
+      modelUuidForRelease = response.body.model.id
+      cy.request('POST', `/api/v2/model/${modelUuidForRelease}/setup/from-schema`, {
+        schemaId: 'minimal-general-v10-beta',
+      }).then((response) => {
+        expect(response.status).to.eq(200)
+        expect(response.body.card).to.have.property('modelId', modelUuidForRelease)
+      })
+    })
+  })
+
+  it('drafts a new release for a model', () => {
+    cy.visit(`${baseURLForRelease}/model/${modelUuidForRelease}`)
+    cy.contains(modelNameForRelease)
+    cy.get('[data-test=modelReleaseTab]').click({ force: true })
+    cy.contains('Draft new Release')
+    cy.get('[data-test=draftNewReleaseButton').click()
+    cy.contains('A release takes a snapshot of the current state of the model code, files and model card')
+
+    cy.get('[data-test=releaseSemanticVersion]').type(releaseVersion, { force: true })
+    cy.get('.w-md-editor-text-input').type('These are some release notes', { force: true })
```

Comment:
Can we use a `data-test` to target this element?

Reply:
If I remember correctly, the actual <input> for the RichTextEditor component is rendered under the hood by the library we use, so the best I found was to use the above class.